### PR TITLE
feat: add error boundaries for jumper

### PR DESCRIPTION
**Pull request by 0xsebn**
Jira: https://lifi.atlassian.net/jira/software/c/projects/LF/boards/15?selectedIssue=LF-5307

### CODE REVIEW

File: src/i18n/en/translation.json [hunk 1]
```diff
@@ -61,6 +61,9 @@
   "button": {
     "okay": "Okay"
   },
+  "error" : {
+    "message": "We encountered a major Error. Please try reloading the page. If the problem persists, contact our support."
```
Comment (chybisov) [inline src/i18n/en/translation.json:65]: Maybe

`Something went wrong. Please try reloading the page. If the problem persists, contact our support.`

Comment (0xsebn) [reply src/i18n/en/translation.json]: Yeah that sounds better

**Comment by dennyscode** [comment]
Nice! Does this get triggered on any 404´s?
Hmm, I am wondering if this will still be needed after the integration of the blog which uses react router at this time of developing it. It comes with some error page.

**Comment by chybisov** [comment]
@dennyscode, it's not about 404 errors. Error boundaries are for rendering errors. Please see here https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary 🙂

**Review by chybisov — APPROVED** [review]

**Review by Addminus — APPROVED** [review]

**Pull request merged**